python/dgl/distributed/partition.py

```diff
@@ -0,0 +1,151 @@
+"""Functions for partitions."""
+
+import json
+import pickle
+import os
+import numpy as np
+
+from .. import backend as F
+from ..base import NID, EID
+from ..data.utils import load_graphs, save_graphs
+from ..transform import metis_partition_assignment, partition_graph_with_halo
+
+def load_partition(conf_file, part_id):
+    ''' Load data of a partition from the data path in the DistGraph server.
+
+    Here we load data through the normal filesystem interface. In the future, we need to support
+    loading data from other storage such as S3 and HDFS.
+
+    Parameters
+    ----------
+    conf_file : str
+        The path of the partition config file.
+    part_id : int
+        The partition Id.
+    '''
+    with open(conf_file) as conf_f:
+        part_metadata = json.load(conf_f)
+    part_files = part_metadata['part-{}'.format(part_id)]
+    node_feats = pickle.load(open(part_files['node_feats'], 'rb'))
+    edge_feats = pickle.load(open(part_files['edge_feats'], 'rb'))
+    client_g = load_graphs(part_files['part_graph'])[0][0]
+    node_map = pickle.load(open(part_metadata['node_map'], 'rb'))
```

Comment:
how about `edge_map`?

Reply:
right now the edge assignment is implicit. Do we want to explicitly store the edge assignment? I think we should support it after we decide to partition the neighborhood of a vertex.